sys/net/gnrc/transport_layer/tcp/gnrc_tcp.c

```diff
@@ -186,8 +186,9 @@ static int _gnrc_tcp_open(gnrc_tcp_tcb_t *tcb, const gnrc_tcp_ep_t *remote,
         tcb->peer_port = remote->port;
 
         /* Setup connection timeout: Put timeout message in TCBs mbox on expiration */
-        _setup_timeout(&connection_timeout, GNRC_TCP_CONNECTION_TIMEOUT_DURATION,
-                       _cb_mbox_put_msg, &connection_timeout_arg);
+        _setup_timeout(&connection_timeout, \
```

Comment:
Here and bellow, please keep the current style of the file for multiline function call. I think you can drop the `\` as well.

Reply:
Okay

Reply:
Done.